color.html

```diff
@@ -75,12 +98,43 @@
       --lumo-success-text-color: hsl(145, 100%, 32%);
       --lumo-success-contrast-color: #FFF;
     }
+    /* MAGI REMOVE START */
   </style>
 </custom-style>
+<!-- MAGI REMOVE END -->
+<!-- MAGI ADD START
+`;
+const $tpl = document.createElement('template');
+$tpl.innerHTML = `<custom-style><style>${colorBase.toString().replace(':host', 'html')}</style></custom-style>`;
+document.head.appendChild($tpl.content);
+
+Vaadin.LumoStyles.colorBase = colorBase;
+})();
+</script>
+
+MAGI ADD END -->
+
+<!-- MAGI ADD START
+<script>
+  /**
+   * @namespace Vaadin
+   */
+  window.Vaadin = window.Vaadin || {};
+
+  /**
+   * @namespace Vaadin.LumoStyles
+   */
+  window.Vaadin.LumoStyles = window.Vaadin.LumoStyles || {};
```

Comment:
Do these really need to be duplicated in every script block in the same file? Wouldn't it be enough to have them defined once at the top? Feels like unnecessary repetition.

Reply:
Thanks, removed.